helm/Chart.yaml

```
@@ -16,14 +16,14 @@ type: application
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
 
-version: 1.1.1
+version: 1.1.2
```

I think this is worth a bigger version bump. Maybe to `1.2.0`
```suggestion
version: 1.2.0
```

---

Sounds good!